<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxCommunications">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Communications</string>
    </property>
    <widget class="QLabel" name="labelSelectSerialPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Serial Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAvailableSerialPort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>71</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>151</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDisconnect">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>151</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBaudRate">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>111</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>9600</number>
     </property>
     <property name="maximum">
      <number>1842300</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>111</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BaudRate:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRefresh">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>71</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Messages</string>
    </property>
    <widget class="QTextEdit" name="textEditMessages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>831</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCommands">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>620</width>
      <height>380</height>
     </rect>
    </property>
    <property name="title">
     <string>Commands</string>
    </property>
    <widget class="QLabel" name="labelChannel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Channel:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxChannel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>33</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>8</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxBiasVoltage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>66</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Bias Voltage</string>
     </property>
     <widget class="QSpinBox" name="spinBoxBiasVoltage">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>34</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelChannel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>38</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelChannel_3">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>65</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Trim:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxTrim">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>63</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonApplyBiasVoltages">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>98</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonInitial">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>33</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>INITIAL</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>66</y>
       <width>150</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Gain</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LNA:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PGA:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxLNAGain">
      <property name="geometry">
       <rect>
        <x>67</x>
        <y>28</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPGAGain">
      <property name="geometry">
       <rect>
        <x>67</x>
        <y>58</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>161</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="title">
      <string>Integrator (HPF)</string>
     </property>
     <widget class="QCheckBox" name="checkBoxPGAIntegrator">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>31</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PGA</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxLNAIntegrator">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>32</y>
        <width>50</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>LNA</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>66</y>
       <width>119</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Active Termination</string>
     </property>
     <widget class="QCheckBox" name="checkBoxActiveTerminationEnable">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>29</y>
        <width>90</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxImpedance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>71</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>46</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Impedance:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonRDFPGA">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>33</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RD FPGA</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>512</x>
       <y>66</y>
       <width>99</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>LPF</string>
     </property>
     <widget class="QComboBox" name="comboBoxLPF">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>66</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>38</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cut-off freq:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonSETCONFIG">
     <property name="geometry">
      <rect>
       <x>413</x>
       <y>33</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SET CONFIG</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonREADCONFIG">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>33</y>
       <width>110</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>READ CONFIG</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>204</y>
       <width>350</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Offset</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Channel offset:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxOffsetVoltage">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>49</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>203</x>
        <y>50</y>
        <width>30</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonApplyChannelOffset">
      <property name="geometry">
       <rect>
        <x>242</x>
        <y>49</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>296</y>
       <width>350</width>
       <height>70</height>
      </rect>
     </property>
     <property name="title">
      <string>V GAIN</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>43</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V GAIN value:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonApplyVGAIN">
      <property name="geometry">
       <rect>
        <x>241</x>
        <y>33</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxVGainValues">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>37</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>43</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>184</y>
       <width>221</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string>ADC Format</string>
     </property>
     <widget class="QCheckBox" name="checkBox2Complement">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>35</y>
        <width>130</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>2' complement</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxOffsetBinary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>58</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Offset binary</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxLSBFirst">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>35</y>
        <width>90</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>LSB First</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxMSBFirst">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>59</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MSB First</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>295</y>
       <width>230</width>
       <height>70</height>
      </rect>
     </property>
     <property name="title">
      <string>Take Multiple Waveforms</string>
     </property>
     <widget class="QCheckBox" name="spinBoxMultipleWaveformsEnable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>41</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMultipleWaveforms">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMultipleWaveformsDirectory">
      <property name="geometry">
       <rect>
        <x>147</x>
        <y>36</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Directory</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelAFE">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>33</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AFE:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAFE">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>33</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Send Raw Command</string>
    </property>
    <widget class="QLineEdit" name="lineEditSendRawCommand">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSendRawCommand">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
